[DOCS-858] Adding log processing rule examples for docker and K8s (#7273)
DataDog/documentation: content/en/agent/kubernetes/log.md

{{% /tab %}}
{{< /tabs >}}

## Advanced log collection

> Use the autodiscovery log annotation to apply an advanced log collection processing logic like:

Reviewer: ```suggestion
Use Autodiscovery log labels to apply advanced log collection processing logic, for example:
```
